src/features/channel-reformat/pin-container.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function() {
+    if (!bttv.settings.get('pinnedHighlights')) {
+        return;
+    }
+
+    // Push pin container to DOM on start if the option is enabled
+    $('.ember-chat .chat-room').append($('<div id="bttv-pin-container">'));
```

Comment:
I could very well see this function caching its result, and returning it, instead of the code relying on this ID.